spdockit-spo/spo-snapshots.md

```diff
@@ -12,6 +12,8 @@ __Please note__ that in order for this feature to work, __you will need to use a
 You can use either [SQL Server database](#internal/configuration/configure-spdockit-database) or a [LocalDB instance](#internal/configuration/configure-localdb) that is shipped with SPDocKit.
 
 Here is what you need to do:
+
+## Manual Snapshot
 1. You can start the loading proces (snapshot) from Backstage. __File > Actions > Take Snapshot__. Or click the __Take Snapshot button__ in __Home ribbon__ of Farm and Site Explorer, Permissions Reports, Content & Usage Reports section, and Snapshots tab.
```

Comment:
Regarding _You can start the loading proces (snapshot) from Backstage._
It should be _the backstage_ (backstage lowercased). Also, users don't usually know what the backstage is, I'd consider maybe some other names? (start page maybe, dunno)

Reply:
we could, if you are interested in changing all 70 mentions of word backstage. (I checked)

Reply:
I'm good, don't worry about it, it's fine :)